package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1039,11 +1039,17 @@ def principal_axes(group, pbc=False):
         """
         atomgroup = group.atoms
         e_val, e_vec = np.linalg.eig(atomgroup.moment_of_inertia(pbc=pbc))
-
+        
```

Comment:
Forgot in my previous review: please add a `.. versionchanged: 1.0.0` entry that states that this function is now guaranteed to return evecs in a R-handed coordinate system. Add a similar sentence to the docs themselves. This is a change that users need to be aware of.

Reply:
Very pedantic, but while you are at it: PEP8 requires blank lines to be empty (line 1042).

Reply:
Added to the doc and sorted line 1042